torchci/pages/api/github/tags/[repoOwner]/[repoName]/[workflow]/[sha].ts

```diff
@@ -0,0 +1,80 @@
+import type { NextApiRequest, NextApiResponse } from "next";
+import { getOctokit, getOctokitWithUserToken } from "lib/github";
+
+function hasWritePermission(permission: string) {
+  return permission === "admin" || permission === "write";
+}
```

Comment:
Feels like an almost a vebatim copy of `hasWritePermission` in `bot/utils.ts`. Can they somehow be refactored together? 
https://github.com/pytorch/test-infra/blob/e35960ffbeaf1a9d048c97a724b4f98422f4528f/torchci/lib/bot/utils.ts#L180